<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="otywin-17" name="Acquisition Scenario" tests="1" failures="1" timestamp="24 Aug 2016 12:14:06 GMT" time="116.778" errors="0">
  <testcase name="start" time="116.758" classname="com.formsdirectinc.functionaltests.apps.TestEngine">
    <failure type="java.io.FileNotFoundException" message="C:\DCIS\14-06\src\test\resources\prepurchaceOutput\paymentsuccess.txt (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: C:\DCIS\14-06\src\test\resources\prepurchaceOutput\paymentsuccess.txt (The system cannot find the file specified)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileReader.<init>(Unknown Source)
	at com.formsdirectinc.functionaltests.tags.AcquisitionUtils.readFile(AcquisitionUtils.java:505)
	at com.formsdirectinc.functionaltests.apps.payment.PaymentSuccess.PaymentSuccesstest(PaymentSuccess.java:110)
	at com.formsdirectinc.functionaltests.apps.payment.Payment.test(Payment.java:328)
	at com.formsdirectinc.functionaltests.apps.Step.executeNextStep(Step.java:29)
	at com.formsdirectinc.functionaltests.apps.TestEngine.start(TestEngine.java:105)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- start -->
</testsuite> <!-- Acquisition Scenario -->
